src/app/models/state.model.ts

```diff
@@ -0,0 +1,13 @@
+import {Recipe} from './recipe.model';
+import {Filters} from './filters.model';
+
+export interface AppState { recipes: { [id: string]: Recipe }, list: string[], filters: Filters };
+export interface State { app: AppState }; // this will also contain router state
+
+export const initialState: State = {
```

Comment:
shouldnt categories be part of state also? I can see they are not in the PR. so we can chat and put them in after as well.